index.sh

```diff
@@ -243,18 +243,29 @@ if [ "${altcmd:-x}" != "x" ] ; then
 |     update-alternatives detected     |
 \\=====================================/
 
+(Notice: getmicro must be run as root, e.x.: curl https://getmic.ro | sudo sh
+ otherwise, trying to y use this feature will fail.)
+ 
+(Notice: if you y use this feature and you put micro in a location unaccessible
```

Comment:
Can we combine this with the note suggesting putting `micro` in `/usr/bin` below?

Reply:
Good idea. I will do this.